tests/E2ETests/test_pytom_FullPipeline.py

```diff
@@ -601,192 +562,75 @@ def test_17_GLocal_Reduced_Binned_CPU(self):
-        if not os.path.exists(outdir3): os.mkdir(outdir3)
-
-        outdir = os.path.join(self.glocaldir, 'alignment_002_cpu')
-        self.outdir_ali2 = outdir
-
-        v = read(f'{self.outdir_ali2}/3-All.em')
-
-        r = resize(v, 2)
-        reference = f'{outdir3}/resizedReference.mrc'
-        write(reference, r)
-
-        cmd = f'cd {self.projectname}/05_Subtomogram_Analysis; mpiexec -n {self.numcores} pytom {self.pytomDir}/bin/GLocalJob.py '
-        cmd += f'--particleList {self.outdir_ali2}/3-ParticleList.xml '
-        cmd += f'--mask {self.refDataDir}/Glocal_mask_200_75_5.mrc '
-        cmd += f'--numberIterations 4 '
-        cmd += f'--pixelSize 2.62 '
-        cmd += f'--particleDiameter 300 '
-        cmd += f'--binning 1 '
-        cmd += f'--destination {outdir3} '
-        cmd += f'--SphericalMask '
-        cmd += f'--angleShells 3 '
-        cmd += f'--angleIncrement 3.00 '
-        cmd += f'--jobName {outdir3}/glocal_input_params_reduced.xml '
-        cmd += f'--reference {reference} '
-
-
-        os.system(cmd)
-
-        from pytom.bin.gen_mask import gen_mask_fsc
-        from pytom.agnostic.io import read, write
-
-        model = sorted([os.path.join(outdir3, f) for f in os.listdir(outdir3) if
+        model = sorted([os.path.join(outdir, f) for f in os.listdir(outdir) if
                         'average-FinalFiltered' in f and f.endswith('em')])[0]
 
         gen_mask_fsc(read(model), 4, f'{self.projectname}/05_Subtomogram_Analysis/Validation/maskFinalAverage.mrc',
                      1, 3)
 
-        # alignedVolume = f'{outdir}/referrenceAligned3-All.em'
-        # alignTwoVolumes(f'{outdir}/3-All.em', self.reference, outname=alignedVolume)
-
         cmd = f'''cd {self.projectname}/05_Subtomogram_Analysis/Validation
 
-fsc.py  '''
-        cmd += f'--v1 {outdir3}/average-Final-Even.em  '
-        cmd += f'--v2 {outdir3}/average-Final-Odd.em '
+        fsc.py  '''
+        cmd += f'--v1 {outdir}/average-Final-Even.em  '
+        cmd += f'--v2 {outdir}/average-Final-Odd.em '
         cmd += f'--mask {self.projectname}/05_Subtomogram_Analysis/Validation/maskFinalAverage.mrc '
         cmd += f'--outputFolder {self.projectname}/05_Subtomogram_Analysis/Validation '
         cmd += f'--fsc 0.143 '
-        cmd += f'--pixelsize 2.62 '
+        cmd += f'--pixelsize 5.24 '
         cmd += f'--randomizePhases 0.000 '
         cmd += f'--combinedResolution '
 
         result = os.popen(cmd).read()
 
         resolution = float(result.split('Resolution determined for pixelsize :')[1].split()[-2])
 
-        self.assertTrue(resolution < 16.,
-                        'Final Resolution of the reconstruction is {resolution}. A resolution below 16. Angstrom is expected.')
+        self.assertTrue(resolution < 11.5,
+                        'Final Resolution of the reconstruction is {resolution}. A resolution below 11.5 Angstrom is '
+                        'expected.')
```

Comment:
This fails for CPU without IMOD (I end with a final resolution of `11.96`)
```suggestion
        self.assertTrue(resolution < 12.0,
                        'Final Resolution of the reconstruction is {resolution}. A resolution below 12.0 Angstrom is '
                        'expected.')
```